src/rdkafka_aux.c

```diff
@@ -277,3 +328,7 @@ const char *rd_kafka_Node_host(const rd_kafka_Node_t *node) {
 uint16_t rd_kafka_Node_port(const rd_kafka_Node_t *node) {
         return node->port;
 }
+
+const char *rd_kafka_Node_rack_id(const rd_kafka_Node_t *node) {
```

Comment:
Better rack only as in J.
```suggestion
const char *rd_kafka_Node_rack(const rd_kafka_Node_t *node) {
```

Reply:
Done